extensions/amp-analytics/0.1/vendors.js

```diff
@@ -77,7 +77,7 @@ export const ANALYTICS_CONFIG = {
       'suffix': '&ref=${documentReferrer}',
       'pageview': '${base}&' +
         'p=${title}&' +
-        's2=${level2}${suffix}',
+        's2=${level2}${extraUrlParams}${suffix}',
```

Comment:
just an observation. You are adding extraUrlParams to the pageview request. Would you not want to add it to the beginning of suffix?